src/main/java/io/confluent/connect/jdbc/dialect/SqliteDatabaseDialect.java

```diff
@@ -129,4 +129,9 @@ public String buildUpsertQueryStatement(
     builder.append(")");
     return builder.toString();
   }
+
+  @Override
+  protected String currentTimestampDatabaseQuery() {
+    return "SELECT strftime('%Y-%m-%d %H:%M:%S.%f','now')";
```

Comment:
Thanks @rhauch for reviewing! 

Based on this suggestion: https://github.com/confluentinc/kafka-connect-jdbc/issues/219#issuecomment-450123435 
The error seemed to be because the sqlite JDBC driver expected a timestamp along with the milliseconds value. 

Retrieving the timestamp as follows fixed this
https://stackoverflow.com/questions/17574784/sqlite-current-timestamp-with-milliseconds